src/applications/vaos/services/appointment/transformers.js

```diff
@@ -593,17 +593,18 @@ export function transformPendingAppointments(requests) {
     const unableToReachVeteran = appt.appointmentRequestDetailCode?.some(
       detail => detail.detailCode?.code === UNABLE_TO_REACH_VETERAN_DETCODE,
     );
+    const created = moment.parseZone(appt.date).format('YYYY-MM-DD');
```

Comment:
We were using two different dates here, but I don't see a need for that. I also don't trust the timezone on the `date` field, so just using the date without time info, since that doesn't matter for EC requests anyway.